### PR TITLE
New Sacct Guide

### LINKED ISSUE #100: Add `sacct` info
I want to have some information about `sacct` in the new SLURM guide. 

- `sacct` is a way to view information about jobs (https://slurm.schedmd.com/sacct.html)
- include recommended options to filter results (`-j`, `--starttime=2020-05-17`, `--endtime=2020-06-19`, `--user=name`)
- talk about `--format=` and provide a list of recommended fields (choose from list - probably nodes, exit codes, etc.) and examples. Also...
- have 1-2 sample queries; one to show how many nodes/cpus a job used, one that prints out which nodes it used and how it exited? Both should have the JobID

### PR DESCRIPTION
**Pull request by EUdds**
Closes #100 
- [X] Documentation Written

- [ ] Sample Output

### CODE REVIEW

File: hpc-job-submission.md [hunk 1]
```diff
@@ -122,7 +122,84 @@ view all jobs for a particular partition with `squeue -p univ`.
...
+Sacct can display different fields about your jobs. You can use the `--helpformat` flag to get a full list.
+
+```	
+[alice@login]$ sacct --helpformat
+```
+{:.term}
+
+### Recommended Fields
+
+When looking for information about your jobs CHTC reccomends using these fields
```
Comment (ChristinaLK) [inline hpc-job-submission.md:186]:
```suggestion
When looking for information about your jobs CHTC recommends using these fields
```

File: hpc-job-submission.md [hunk 2]
```diff
@@ -122,7 +122,84 @@ view all jobs for a particular partition with `squeue -p univ`.
...
+nnodes
+nodelist
+ntasks
+partition
+start
+state
+submit
+user
+```
+
```
Comment (ChristinaLK) [inline hpc-job-submission.md:202]: Can you add something here like "a sample `sacct` command that prints out when a job started, ended, and how many nodes and cpus it requested would like like this:" and then have an example?

File: hpc-job-submission.md [hunk 3]
```diff
@@ -122,7 +122,84 @@ view all jobs for a particular partition with `squeue -p univ`.
...
+	
+	```
+	[alice@login]$ sacct -X
+	```
+	{:.term}
+	<!-- Sample Output -->
+
+## Displaying Specific Fields
+
+Sacct can display different fields about your jobs. You can use the `--helpformat` flag to get a full list.
```
Comment (ChristinaLK) [inline hpc-job-submission.md:177]:
```suggestion
`sacct` can display different fields about your jobs. You can use the `--helpformat` flag to get a full list.
```

**Comment by EUdds** [comment]
Added some more example commands @ChristinaLK

File: hpc-job-submission.md [hunk 4]
```diff
@@ -122,7 +122,99 @@ view all jobs for a particular partition with `squeue -p univ`.
...
+
+```
+[alice@login]$ sacct --start=2018-01-01
+```
+{:.term}
+
+
+## How To Select Jobs
+
+- To display information about a specific job or list of jobs use `-j` or `--jobs` Followed by a job number or comma seperated list of job numbers.
```
Comment (ChristinaLK) [inline hpc-job-submission.md:137]:
```suggestion
- To display information about a specific job or list of jobs use `-j` or `--jobs` followed by a job number or comma separated list of job numbers.
```

File: hpc-job-submission.md [hunk 5]
```diff
@@ -122,7 +122,99 @@ view all jobs for a particular partition with `squeue -p univ`.
...
+	
+	```
+	[alice@login]$ sacct -jobs job1,job2,job3
```
Comment (ChristinaLK) [inline hpc-job-submission.md:140]:
```suggestion
	[alice@login]$ sacct --jobs job1,job2,job3
```

File: hpc-job-submission.md [hunk 6]
```diff
@@ -122,7 +122,99 @@ view all jobs for a particular partition with `squeue -p univ`.
...
+	```
+	{:.term}
+<!-- Sample output -->
+
+- To select information about jobs in a certian date range use `--start` and `--end` Without it, `sacct` will only return jobs from the current day.
```
Comment (ChristinaLK) [inline hpc-job-submission.md:145]:
```suggestion
- To select information about jobs in a certain date range use `--start` and `--end` Without it, `sacct` will only return jobs from the current day.
```

**Pull request merged**